internal/commands/request/request_internal.go

```diff
@@ -57,6 +57,11 @@ func runInternalPingOneRequest(uri string) (err error) {
 		return err
 	}
 
+	failOption, err := getFailOption()
```

Comment:
We need to handle the error here anyways, so this function is unneeded. `profiles.GetOptionValue` returns the exact same as `getFailOption`.